packages/core/demo/data/bar.ts

```diff
@@ -478,6 +478,25 @@ export const simpleBarSkeletonOptions = {
 	}
 };
 
+// simple bar - error
+export const simpleBarErrorData = [];
+export const simpleBarErrorOptions = {
+	title: "Simple bar (error)",
+	axes: {
+		left: {},
+		bottom: {
+			scaleType: "labels"
+		}
+	},
+	data: {
+		loading: false,
+		error: {
+			title: "No data available",
+			subtitle: "Lorem ipsum dolor sit."
```

Comment:
Maybe we should use a real subtitle here to make for a better demo? e.g. `Unable to reach application servers at this time`